pkg/kubelet/kubelet.go

```diff
@@ -1850,6 +1850,12 @@ func (kl *Kubelet) syncLoopIteration(configCh <-chan kubetypes.PodUpdate, handle
 			}
 		}
 
+		if e.Type == pleg.ContainerStarted {
```

Comment:
remind me where `<-plegCh` events come from? can this still race with a DELETE event from `<-configCh`?

Reply:
They come from here
https://github.com/sjenning/kubernetes/blob/a104fa36d24d95a6cc87889b16e49e4f76abb59c/pkg/kubelet/pleg/generic.go#L188-L302

At first I thought I had just created a loop and that the runtimeCache fed PLEG, but that isn't true.

PLEG calls kuberuntime's `GetPods()` (different from the `runtimeCache`), which calls directly into the actual runtime service via `getKubeletSandboxes`/`getKubeletContainers`.